colossalai/inference/core/engine.py

```diff
@@ -15,36 +18,25 @@ class InferEngine:
         - Log the generation process
 
     Args:
+        tokenizer: Path of the tokenizer to use.
         colossal_config: We provide a unified config api for that wrapped all the configs. You can use it to replace the below configs.
-        model_config : The configuration for the model.
-        parallel_config: The configuration for parallelize model.
-        cache_config : Configuration for initialize and manage kv cache.
-        tokenizer (Tokenizer): The tokenizer to be used for inference.
         use_logger (bool): Determine whether or not to log the generation process.
     """
 
     def __init__(
         self,
-        model_config,
-        cache_config,
-        parallel_config,
-        tokenizer,
-        use_logger: bool = False,
+        tokenizer: str = "",
         colossal_config: Optional["ColossalInferConfig"] = None,
+        use_logger: bool = False,
```

Comment:
This argument should be `verbose` by convention.

Reply:
Done